.storybook/recipes/MenuWithIconButton.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+import React from 'react';
+
+import { Icon } from '../../src';
+import { Menu, type MenuProps } from '../../src/components/Menu/Menu';
+import icons from '../../src/icons/spritemap';
+
+export default {
+  title: 'Recipes/MenuWithIconButton',
+  component: Menu,
+  parameters: {
+    badges: [BADGE.BETA],
+    layout: 'centered',
+  },
+  argTypes: {
+    iconName: {
+      control: 'radio',
+      options: Object.keys(icons),
+      defaultValue: 'dots-vertical',
+    },
+  },
+} as Meta<MenuProps>;
+
+export const WithVerticalDotsButton: StoryObj<MenuProps> = {
+  render: ({ iconName, args }) => (
+    <Menu {...args}>
+      <Menu.PlainButton className="hover:bg-button-secondary-neutral-background-hover">
```

Comment:
hmm I feel like it'd be worth adding a hover color & border radius to the plain button styling. those don't feel too opinionated and reduce a bit of developer hassle.

Reply:
Ooh agree with border radius, but not sure about hover color since it's a transparent background, will ask design

Reply:
Ohhh I agree with border radius but will ask design about hover bg color since the default bg is transparent and the hover tokens we have are solid colors